packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -272,6 +273,20 @@ const ruleset: any = {
     /// ARM RPC rules for Patch patterns
     ///
 
+    // RPC Code: RPC-Patch-V1-01
+    PatchPropertiesCorrespondToPutProperties: {
+      description:
+        "Patch request body MUST contain at least one or more properties present in the normal resource definition (PUT operation).",
+      message: "{{error}}",
+      severity: "error",
```

Comment:
> severity: "error", [](http://example.com/codeflow?start=5&length=19)

set staging: true

Reply:
[https://github.com/Azure/azure-openapi-validator/blob/main/CONTRIBUTING.md#how-to-set-a-spectral-rule-to-run-only-in-staging](https://github.com/Azure/azure-openapi-validator/blob/main/CONTRIBUTING.md#how-to-set-a-spectral-rule-to-run-only-in-staging)

Reply:
setting stagingOnly: true is causing build errors locally/ tests fail.
Also, I dont find any rule have this flag enabled & I believe all the rules when merged to main by default run only in Staging.

Reply:
We do have rules merged into main with `stagingOnly` true, e.g., https://github.com/Azure/azure-openapi-validator/blob/main/packages/rulesets/src/spectral/az-arm.ts#L163. What error do you get in the tests when you add `stagingOnly`?

Reply:
Added it!
thanks